web/src/main/resources/sample_categories.xml

```diff
@@ -12,6 +12,9 @@
     <Category>
         <name>DASHBOARD</name>
     </Category>
+    <Category>
+        <name>BACKGROUND_THUMBNAIL</name>
```

Comment:
this is not needed since the background thumbnail is  saved in map config. 
please double and triple check this in the PR because it is important to push only relevant changes